### PR TITLE
Erlang 22 / Elixir 1.9 support

### PR DESCRIPTION
**Pull request by lboyarsky**
Ticket: [update API to Erlang 22, Elixir 1.9](https://app.asana.com/0/810933294009540/1129956922358338/f)

Relevant changes are in last two commits.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /deps
 erl_crash.dump
 *.ez
+.elixir_ls/
```
Comment (paulswartz) [inline .gitignore]: should have a trailing newline to avoid an extraneous diff later on

File: .travis.yml [hunk 2]
```diff
@@ -2,26 +2,11 @@ dist: trusty
 language: elixir
 
 elixir:
-    - 1.2.6
-    - 1.3.4
-    - 1.4.1
-    - 1.5.1
     - 1.6.1
+    - 1.7.4
```
Comment (paulswartz) [inline .travis.yml:6]: Should we add 1.8 and 1.9 since that's what the PR is updating?

File: .travis.yml [hunk 3]
```diff
@@ -2,26 +2,11 @@ dist: trusty
...
 
 otp_release:
-    - 18.3
     - 19.0
     - 20.0
```
Comment (paulswartz) [inline .travis.yml:12]: Add 21.3 and 22.0?

File: .gitignore (hunk 1 above)
Comment (lboyarsky) [reply .gitignore]: Fixed

File: .travis.yml (hunk 2 above)
Comment (lboyarsky) [reply .travis.yml:6]: Done, but it's about their CI system building multiple versions in parallel - https://travis-ci.org/xerions/phoenix_swagger/builds/561320905 - I'm not sure if there is any use for us.

File: .travis.yml (hunk 3 above)
Comment (lboyarsky) [reply .travis.yml:12]: Done, same as above.

File: .travis.yml (hunk 2 above)
Comment (paulswartz) [reply .travis.yml:6]: If we want the changes to go back upstream, then we should probably test that they're working with the new versions.

**Review by paulswartz — APPROVED** [review]
🍰 and then the API ticket can be updated to point at `master`.

**Pull request merged**